packages/blobs/desktop/lightdm/slick-greeter.conf

```diff
@@ -0,0 +1,11 @@
+[Greeter]
+background = /usr/share/backgrounds/armbian/armbian03-Dre0x-Minum-dark-blurred-3840x2160.jpg
```

Comment:
This parameter is somehow ignored.

Reply:
Are you sure? If I comment it, I just see a black background in the greeter.
According to their readme, it is a valid option: https://github.com/linuxmint/slick-greeter/blob/master/README.md

Reply:
Its wallpaper, but not this blurred one. As it was before. Something is not right but dunno what.

Reply:
If possible we would like to keep all things we had before. Not a black screen, but different wallpaper - blured for lock and not blured for normal operations.

Reply:
I'm afraid I'm not understanding :frowning_face: 

The current 'background = /usr/share/backgrounds/armbian/armbian03-Dre0x-Minum-dark-blurred-3840x2160.jpg' is the same as lightdm-gtk-greeter.conf and shows the blurred wallpaper for the greeter/lock screen, while the regular desktop has the usual non-blurred wallpaper.

It works this way on my installations (two armhf, one arm64 which is also working fine with lightdm-gtk-greeter, but tried for fun), but will double check and report

Reply:
> the same as lightdm-gtk-greeter.conf and shows the blurred wallpaper for the greeter/lock screen

Here I don't have that ... perhaps it was some other trouble. I only tested a build made by @RichNeese and it was not blured. OK, I'll also check again later on. It could be some other problem. Sorry for the noise.

Reply:
No problem ;) I'll double check later too